inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -255,19 +309,19 @@ public function upload_directory( $args, $args_assoc ) {
 		require_once dirname( __FILE__ ) . '/class-s3-uploads-changed-files-iterator.php';
 
 		try {
-			$s3->uploadDirectory( 
```

Comment:
In general, avoid unrelated whitespace changes, since they can cause merge conflicts.